.gitignore

```diff
@@ -349,3 +349,6 @@ MigrationBackup/
 
 # Ionide (cross platform F# VS Code tools) working folder
 .ionide/
+
+# Custom
+docs/_site/
```

Comment:
Wow, this is maybe a big nitpick: I think there's some places in the `.gitignore` where docs are maybe placed together.

Reply:
Removing it, no longer required AFAIK